connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/PluginUtils.java

```diff
@@ -197,15 +202,23 @@ public static boolean isClassFile(Path path) {
         return path.toString().toLowerCase(Locale.ROOT).endsWith(".class");
     }
 
-    public static Set<Path> pluginLocations(String pluginPath) {
+    public static Set<Path> pluginLocations(String pluginPath, boolean failFast) {
         if (pluginPath == null) {
             return Collections.emptySet();
         }
         String[] pluginPathElements = COMMA_WITH_WHITESPACE.split(pluginPath.trim(), -1);
         Set<Path> pluginLocations = new LinkedHashSet<>();
         for (String path : pluginPathElements) {
             try {
-                Path pluginPathElement = Paths.get(path).toAbsolutePath();
+                Path specifiedPath = Paths.get(path);
+                Path pluginPathElement = specifiedPath.toAbsolutePath();
+                if (!specifiedPath.isAbsolute()) {
+                    log.warn("Plugin path element '{}' is relative, evaluating to {}.",
+                            specifiedPath, pluginPathElement);
+                }
```

Comment:
I don't think we need to warn people when any relative path is specified. The only case I think we need to warn about is when an empty (but non-null) path is given. The way relative paths work is fairly intuitive; the way Connect workers and this CLI work with an empty path is less so (and is more likely to come from user error).

```suggestion
                Path pluginPathElement = Paths.get(pluginPath).toAbsolutePath();
                if (pluginPath.isEmpty()) {
                    log.warn("Plugin path element is empty, evaluating to current directory {}", pluginPathElement);
                }
```

Reply:
The condition here should be `pluginPath.isEmpty()`, not `!pluginPath.isEmpty()`.